docs/index.rst

```diff
@@ -40,15 +66,46 @@ going straight to the :doc:`End User Topics <end-user-topics/end-user-topics>` o
    getting-started/getting-started-with-services
    getting-started/github-apps/github-apps-landing-page
 
-.. toctree::
-   :caption: Videos (Tutorials & Presentations)
-   :maxdepth: 1
+Launching tools and workflows
+-----------------------------
+Multi-platform: :doc:`AnVIL <launch-with/anvil-launch-with>` | :doc:`BioData Catalyst <launch-with/bdcat-launch-with>` | :doc:`Dockstore CLI <launch-with/launch>`
```

Comment:
You mean for `Multi-platform` to be the currently selected item, right?

If so, the colon is throwing me off. It looks like AnVIL, BD Cat and Dockstore CLI are the possible multi-platforms, when I think you are referring to the logos below.

Reply:
AnVIL, BDC, and Dockstore CLI are indeed intended to the be the multi-platform launchers, since all of them can launch on multiple platforms. The logos below are platform-specific.

Reply:
This section doesn't quite click for me. But could just be me... The relationship between these links and the logos below is not clear, although I get it's tricky because you can't use the BD Cat logo.

![Screen Shot 2022-02-22 at 1 11 14 PM](https://user-images.githubusercontent.com/1049340/155220169-8ab2630d-b2ee-4b89-be5f-3c7659b69998.png)